README.md

```diff
@@ -424,7 +426,48 @@ Assumes database resolution is done by adding prefixes. If database resolution i
      where h.date = '2016-05-13'
        and h.hour = 1
     ;
+
+## Database Name Mapping
+Waggle Dance allows configuration to map database names. This feature is intended to be useful when migrating from an existing database into different environments.
+A use case:
+We have legacy data lakes that have tables belonging to the same domain in differently named databases. E.g. a booking domain is called 'datawarehouse' or 'hotel_booking'. When federating to those datalakes it helps data discovery if those names are standardized to one name. Renaming a database is easy but migrating ETL might not be, so to make migration easier you can configure Waggle Dance to remap the names.
+
+Example Datalake X federating to 2 other "legacy" datalakes Y and Z.
+This is what we want the different databases to look like in the datalakes:
+
+X has:
+
+    y_booking
+    z_booking
+
+Y has:
+    
+    datawarehouse
+    another_db
+
+Z has:
     
+    hotel_booking
+
+To achieve this we can configure Waggle Dance in X to map the names:
+
+
+    federated-meta-stores:
+      - remote-meta-store-uris: thrift://10.0.0.1:9083
+        name: y
+        mapped-databases:
+        - datawarehouse
+        database-name-mapping:
+          datawarehouse: booking
+      - remote-meta-store-uris: thrift://10.0.0.2:9083
+        name: z
+        database-name-mapping:
+          hotel_booking: booking
+
+
```

Comment:
This now lets you refer to the above referenced databases in scripts as either "y_datawarehouse" or "y_booking" and "z_hotel_booking" or "z_booking".

Reply:
I'm correct in that you can use both the old and new names right?

Not sure whether it would be useful to show an example bit of SQL demonstrating before and after to make this all a bit clearer?

Reply:
you cannot use the old name e.g. hotel_booking. Only the "mapped" name is exposed. But that's because you're looking through Waggle Dance tinted glasses.

Reply:
Ah OK, so some of my suggestions might be incorrect then, please ignore/update them accordingly.